include/tvm/relay/transform.h

```diff
@@ -274,6 +274,52 @@ class Sequential : public Pass {
   using ContainerType = Sequential;
 };
 
+/*!
+ * \brief A class to save registered passes.
+ */
+class PassRegistry {
```

Comment:
We should not have this, we just need to use function registry to register pass constructors

Reply:
registered under transform.PassName

Reply:
I thought we may need to this to help pass analysis or validation on the registered passes. I can remove it for now.